homeassistant/bootstrap.py

```diff
@@ -504,7 +504,8 @@ def async_from_config_file(config_path: str,
     try:
         config_dict = yield from hass.loop.run_in_executor(
             None, conf_util.load_yaml_config_file, config_path)
-    except HomeAssistantError:
+    except HomeAssistantError as err:
+        _LOGGER.exception(err)
```

Comment:
We should log with `_LOGGER.error` and we should add a message around it. `"Error loading %s: %s", config_path, err`